target/linux/brcm47xx/image/Makefile

```diff
@@ -400,9 +400,9 @@ define Device/linksys-wrt54g3gv2-vf
   DEVICE_MODEL := WRT54G3GV2-VF
   DEVICE_PACKAGES := kmod-b43 $(USB2_PACKAGES)
   FILESYSTEMS := $(FS_128K)
-  IMAGES := noheader.bin bin
-  IMAGE/noheader.bin := linksys-pattern-partition | append-rootfs | trx-v2-with-loader
-  IMAGE/bin := linksys-pattern-partition | append-rootfs | trx-v2-with-loader | linksys-bin
+  IMAGES := noheader-factory.bin factory.bin
+  IMAGE/noheader-factory.bin := linksys-pattern-partition | append-rootfs | trx-v2-with-loader
+  IMAGE/factory.bin := linksys-pattern-partition | append-rootfs | trx-v2-with-loader | linksys-bin
```

Comment:
So this is an example where IMO one should look at what the images are good for. So one could name them after their _use_ (different ways of flashing?) and not after their _content_.